pkg/config/log.go

```diff
@@ -111,21 +118,38 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 			}
 		}
 		seelogConfig.ConfigureSyslog(syslogURI, useTLS)
+		jmxSeelogConfig.ConfigureSyslog(syslogURI, useTLS)
```

Comment:
I think `setupLogger` should not change the two loggers simultaneously.
Either you implement two function (e.g. `setupJMXLogger`& `setupMainLogger` and make them call `setupLogger` that will contain common code and deal with the difference in the two wrappers.